<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>human_translation</class>
 <widget class="QMainWindow" name="human_translation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>784</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>仿宋</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>翻译系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.gif</normaloff>img/icon.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文档翻译</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>单词本</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>415</y>
      <width>750</width>
      <height>330</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>750</width>
      <height>330</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="translation">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>翻 译</string>
    </property>
   </widget>
   <widget class="QPushButton" name="human_translation_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>人工翻译</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>中英文翻译系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>logo</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgcolor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>831</width>
      <height>791</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>logo1</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>logo1</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>logo1</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>logo1</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>logo5</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgcolor1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>245</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dictionary1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>词典</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dictionary2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>翻译</string>
    </property>
   </widget>
   <zorder>bgcolor1</zorder>
   <zorder>bgcolor</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>textBrowser</zorder>
   <zorder>textEdit</zorder>
   <zorder>translation</zorder>
   <zorder>human_translation_2</zorder>
   <zorder>label_5</zorder>
   <zorder>logo</zorder>
   <zorder>logo1</zorder>
   <zorder>logo4</zorder>
   <zorder>logo3</zorder>
   <zorder>logo5</zorder>
   <zorder>logo2</zorder>
   <zorder>dictionary1</zorder>
   <zorder>dictionary2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
